src/EnergyPlus/HeatingCoils.cc

```diff
@@ -984,38 +984,155 @@ namespace HeatingCoils {
 
             TestCompSet(CurrentModuleObject, Alphas(1), Alphas(3), Alphas(4), "Air Nodes");
 
+            if ((UtilityRoutines::SameString(Alphas(5), "Refrigeration:Condenser:AirCooled")) ||
+                (UtilityRoutines::SameString(Alphas(5), "Refrigeration:Condenser:EvaporativeCooled")) ||
+                (UtilityRoutines::SameString(Alphas(5), "Refrigeration:Condenser:WaterCooled"))) {
+                if (lNumericBlanks(1)) {
+                    HeatingCoil(CoilNum).Efficiency = 0.8;
+                } else {
+                    HeatingCoil(CoilNum).Efficiency = Numbers(1);
+                    if (Numbers(1) < 0.0 || Numbers(1) > 0.9) {
+                        ShowSevereError(CurrentModuleObject + ", \"" + HeatingCoil(CoilNum).Name +
+                                        "\" heat reclaim recovery efficiency must be >= 0 and <=0.9");
+                        InputErrorsFound = true;
+                    }
+                }
+            } else {
+                if (lNumericBlanks(1)) {
+                    HeatingCoil(CoilNum).Efficiency = 0.25;
+                } else {
+                    HeatingCoil(CoilNum).Efficiency = Numbers(1);
```

Comment:
Move this part ahead to use efficiency number later to check total reclaim efficiency (by multiple users, from the same source coil) not exceeding limitation.